go/vt/sqlparser/parse_test.go

```diff
@@ -1927,7 +1927,10 @@ var (
 	}, {
 		input: "show vschema vindexes",
 	}, {
-		input: "show vschema vindexes on t",
+		input: "show vschema vindexes from t",
+	}, {
+		input:  "show vschema vindexes on t",
+		output: "show vschema vindexes from t",
```

Comment:
Shouldn't we add tests for the new show commands here as well? Such as `show vitess_targets`, `show vitess_metadata variables`, etc.

Reply:
these are not new commands. but I can add.